apps/sensenet/src/services/CommandProviders/CheatCommandProvider.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@furystack/inject'
+import { CommandPaletteItem } from '../../store/CommandPalette'
+import { CommandProvider } from '../CommandProviderManager'
+
+@Injectable()
+export class CheatCommandProvider implements CommandProvider {
+  public items: Record<string, CommandPaletteItem> = {
+    iddqd: {
+      primaryText: 'GOD MODE ON',
+      secondaryText: '',
+      url: 'https://classicreload.com/dosx-doom.html',
+      content: {
+        Id: 0,
+        Path: '',
+        Name: '',
+        Type: 'DoomCheatContent',
+      },
+    },
+    'pot of gold': {
+      primaryText: 'cheat enabled you wascally wabbit',
+      secondaryText: 'now take a screenshot and go talk to your line manager',
+      url: 'https://www.myabandonware.com/game/warcraft-orcs-humans-250/play-250',
```

Comment:
This returns a 404

Reply:
it was just released on [GoG](https://www.gog.com/game/warcraft_bundle) - but this URL is unused at the moment. We should create a component for old DOS games in the future :)